### DISCUSSION TITLE
Updating node-zwave-js to 12.9.1

### DISCUSSION THREAD

**Discussion by Hooverjc** [zwave-js/zwave-js-ui#3727] [General]
Looks like we are currently about a month and 6 releases behind here. Was just curious if updating to the current release had a schedule behind it or if there was something else holding it up, I appreciate the continued work on this!

**Comment by Botched1** [comment]
I would assume zniffer support is taking a lot of time. But that's just a guess.

**Reply by robertsLando** [reply]
Tooks more then 3 weeks but now it's ready and I'm going to release it today: https://github.com/zwave-js/zwave-js-ui/pull/3706

Sorry for the delay but it has been a pain

**Reply by Botched1** [reply]
No apologies needed. You're doing great, and it is perfectly reasonable that a large new feature would take a commiserate amount of time!

**Reply by robertsLando** [reply]
Thanks  for understanding 🙏🏼❤️

**Reply by Hooverjc** [reply]
> Tooks more then 3 weeks but now it's ready and I'm going to release it today: #3706
> 
> Sorry for the delay but it has been a pain

Agreed, no need to apologize, I hadn't seen the PR for that feature and connected the dots after Botched1 called it out. I appreciate it!
